webpack.config.js

```diff
@@ -1,58 +1,111 @@
+  resolve: {
+    alias: {
+      svelte: path.dirname(require.resolve("svelte/package.json")),
     },
-    module: {
-        rules: [
-            {
-                test: /\.tsx?$/,
-                use: 'ts-loader',
-                exclude: /node_modules/,
+    extensions: [".tsx", ".ts", ".js", ".svelte"],
+    mainFields: ["svelte", "browser", "module", "main"],
+  },
+  optimization: {
+    minimize: true,
+    minimizer: ["...", new CssMinimizerPlugin({ parallel: true })],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: [
+          {
+            loader: "ts-loader",
+            options: {
+              transpileOnly: true,
             },
-            {
-                test: /\.svelte$/,
-                use: {
-                    loader: 'svelte-loader',
-                    options: {
-                        emitCss: true,
-                        hotReload: true
-                    }
-                },
-                exclude: /node_modules/
+          },
+        ],
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.svelte$/,
+        use: {
+          loader: "svelte-loader",
+          options: {
+            compilerOptions: {
+              dev: false,
+            },
+            emitCss: true,
+            hotReload: false,
+          },
+        },
+      },
+      {
+        test: /\.css$/,
+        use: [
+          "style-loader",
+          {
+            loader: "css-loader",
+            options: {
+              sourceMap: !prod,
+              url: false, // necessary if you use url('/path/to/some/asset.png|jpg|gif')
             },
-            {
-                test: /\.css$/,
-                use: [
-					/**
-					 * MiniCssExtractPlugin doesn't support HMR.
-					 * For developing, use 'style-loader' instead.
-					 * */
-                    prod ? MiniCssExtractPlugin.loader : 'style-loader',
-                    'css-loader'
-                ]
-            }
+          },
+          {
+            loader: "postcss-loader",
+            options: {
+              postcssOptions: {
+                plugins: [
+                  tailwindcss(path.join(__dirname, "tailwind.config.js")),
+                  require("autoprefixer"),
+                ],
+              },
+            },
+          },
         ],
-    },
-    output: {
-        filename: 'code.js',
-        path: path.resolve(__dirname, 'public'),
-    },
-    mode,
-    plugins: [
-        new MiniCssExtractPlugin({
-            filename: '[name].css'
-        })
+      },
     ],
-};
\ No newline at end of file
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      global: {}, // Fix missing symbol error when running in developer VM
+    }),
+    new HtmlWebpackPlugin({
+      inject: "body",
+      template: "./src/template.html",
+      filename: "index.html",
+      chunks: ["ui"],
+      cache: false,
+    }),
+    new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/ui/]),
```

Comment:
this inlines the ui.js file into index.html since figma does not allow importing javascript from external files